app/views/announcements/index.html.erb

```diff
@@ -1,14 +1,14 @@
 <div class="container">
   <div class="row center-row">
-    <div class="col-xs col-sm col-md col-lg">
+    <div class="col-xs col-sm col-md col-lg an-lg-green">
```

Comment:
Provide a meaningful class name instead of `an-lg-green`
ig reviewdog maybe failing for this reason

Reply:
some things wrong with this PR i am opening a new one #4288